apps/meteor/client/views/marketplace/AppsProvider.tsx

```diff
@@ -322,6 +375,14 @@ const AppsProvider: FC = ({ children }) => {
 					return;
 				}
 
+				if (app.private) {
+					dispatchPrivateApps({
+						type: 'delete',
+						appId,
+						reload: fetch,
+					});
```

Comment:
No `return`? It's supposed to dispatch twice?

Reply:
two different lists